doc/dev/crimson/crimson.rst

```diff
@@ -141,8 +141,39 @@ You could stop the vstart cluster using::
 
   $ ../src/stop.sh --crimson
 
+Metrics and Tracing
+===================
 
-CBT Based Testing
+Crimson offers three ways to report the stats and metrics
```

Comment:
nit, missing "."

Reply:
i am adding a ":" instead. updated and repushed.